src/app/todos/page.js

```diff
@@ -37,8 +37,30 @@ export default function ToDos() {
 
     function removeTodo({ index }) {
         setTodos(todos.filter((v,idx) => idx!==index));
```

Comment:
move this into the chained then so it will only update the front end on successful calback